src/pdf.coffee

```diff
@@ -90,17 +91,21 @@ module.exports = class PDF
       child.kill()
 
     child.on 'exit', (code) ->
-      clearTimeout(timeout)
-      if code || stderr.length
-        err = new Error(Buffer.concat(stderr).toString() or 'html-pdf: Unknown Error')
-        return callback(err)
-      else
-        try
-          data = Buffer.concat(stdout).toString()?.trim()
-          data = JSON.parse(data)
-        catch err
+      # Wait X milliseconds after the 'exit' event being fired
+      # as 'data' event might still be processing
+      phantomExitTimeout = (options.phantomExitTimeout || 60)
+      setTimeout ->
```

Comment:
I'm sorry. In the link you provided they work around the problem that `console.log()` writes to stdout in the next tick. We don't have that issue in here. You probably ran into an a timing issue in the phantomjs process. What's your use case?